spec/controllers/equipment_models_controller_spec.rb

```diff
@@ -19,42 +19,34 @@
         it_behaves_like 'successful request', :index
       end
 
-      context 'with @category set' do
-        it 'populates an array of of active category-type equipment models' do
+      it 'defaults to all active equipment models' do
```

Comment:
I think it's less clear because I use the `stub_collection_methods` method instead of an explicit mock (`allow(EquipmentModel).to receive(:all).and_return(models)`). We're expecting that `models` receives `includes` and `active`. I could change the stub on `EquipmentModel` to an expectation to make things more explicit?